spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/firestore/GcpFirestoreAutoConfigurationTests.java

```diff
@@ -95,6 +96,20 @@ void testTransactionManagerExcludedWithoutAutoConfiguration() {
             });
   }
 
+  @Test
+  void testDatabaseId() {
```

Comment:
Maybe this test can be named a little more specific? The above test is testing the default behavior, so this one can be something like `testDatabaseIdOverride`. 
See also [unit test naming](https://engdoc.corp.google.com/eng/doc/devguide/testing/unit/best-practices.md?cl=head&polyglot=java#naming), I know we don't have this naming convention in Spring Cloud GCP yet, so we don't have to strictly follow it, the idea is good though to include the behavior and outcome in the test name.